src/Node.ts

```diff
@@ -1,50 +1,64 @@
 import { fabric } from "fabric";
 import { makeLine } from "./Utils";
+import { Pointer } from "./Pointer";
 import Config = require("./Config");
 
-
 export class Node {
     public data: number;
-    public next: Node;
-    private representation: fabric.Object;
-    
-    constructor(data: number, next: Node) {
+    public next: Pointer;
+    private representation: fabric.Group;
```

Comment:
So glad you figured this out. I was wondering if Fabric had a good way to group objects together.